wp-content/themes/core/components/blocks/post_list/Post_List_Controller.php

```diff
@@ -85,28 +87,58 @@ public function get_posts_card_args() {
 		$cards = [];
 		foreach ( $this->posts as $post ) {
 			$link    = $post->get_link();
+			$uuid    = uniqid( 'p-' );
 			$cards[] = [
-				Card_Controller::TITLE   => defer_template_part(
+				Card_Controller::STYLE           => Card_Controller::STYLE_ELEVATED,
+				Card_Controller::USE_TARGET_LINK => true,
+				Card_Controller::TITLE           => defer_template_part(
 					'components/text/text',
 					null,
 					[
+						Text_Controller::TAG     => 'h3',
+						Text_Controller::CLASSES => [ 'h5' ],
 						Text_Controller::CONTENT => $post->get_title(),
+						// Required for screen reader accessibility, below.
+						Text_Controller::ATTRS   => [ 'id' => $uuid . '-title' ],
 					]
 				),
-				Card_Controller::CONTENT => defer_template_part(
-					'components/text/text',
+				Card_Controller::DESCRIPTION     => defer_template_part(
+					'components/container/container',
 					null,
 					[
-						Text_Controller::CONTENT => $post->get_excerpt(),
-					]
+						Container_Controller::CONTENT => wpautop( $post->get_excerpt() ),
+						Container_Controller::CLASSES => [ 't-sink', 's-sink' ],
+					],
+				),
+				Card_Controller::IMAGE           => defer_template_part(
+					'components/image/image',
+					null,
+					[
+						Image_Controller::IMG_ID       => $post->get_image_id(),
+						Image_Controller::AS_BG        => true,
+						Image_Controller::CLASSES      => [ 'c-image--bg', 's-aspect-4-3' ],
+						Image_Controller::SRC_SIZE     => Image_Sizes::FOUR_THREE,
+						Image_Controller::SRCSET_SIZES => [
+							Image_Sizes::FOUR_THREE,
+							Image_Sizes::FOUR_THREE_SMALL,
+						],
+					],
 				),
-				Card_Controller::IMAGE   => $post->get_image_id(),
-				Card_Controller::CTA     => defer_template_part(
+				Card_Controller::CTA             => defer_template_part(
 					'components/link/link',
 					null,
 					[
-						Link_Controller::CONTENT => $link[ 'label' ] ?? $link[ 'url' ],
-						Link_Controller::URL     => $link[ 'url' ],
+						Link_Controller::CONTENT => __( 'Read More', 'tribe' ),
+						Link_Controller::URL     => $link['url'],
+						Link_Controller::CLASSES => [ 'a-cta', 'is-target-link' ],
+						Link_Controller::ATTRS   => [
+							// These attrs provide the most screen reader accessible link.
+							'id'               => $uuid . '-link',
+							'aria-labelledby'  => $uuid . '-title',
+							'aria-describedby' => $uuid . '-link',
+							// Sets this link as the card's click-within target link.
+							'data-js'          => 'target-link',
+						],
```

Comment:
Nothing to change, just thinking out loud...There is enough needed for this particular type of card from the a11y front + link functionality piece that especially if we go the card component per card variation route, it might make more sense to set these as defaults vs needing to declare them every time.

One other thought regarding the target linking, to make it simpler vs. needing to also define the things at every level, I was considering turning that into more of a card level bool argument that then flips on/off the right classes/etc. for the whole card. Looks like you have the bool already and are using it that way to some degree. Thoughts about leveraging it to rely 100% on the arg?

Reply:
I thought about that as well. My concern is that a card could not use the CTA field and/or have another link (such as the post title) that should be used as the target link. In the spirit of keeping the low-level components as flexible as possible, I think it makes more sens to let the usage dictate any application of the values/settings and options.